templates/partials/introduction/basic.html

```diff
@@ -1,7 +1,19 @@
+{% from "components/panel/_macro.njk" import onsPanel %}
+
 <div id="{{content_block.id}}" class="ons-u-mb-s">
   {% if content_block.title %}
     <h2>{{ content_block.title }}</h2>
   {% endif %}
   {% set contents = content_block.contents %}
   {% include 'partials/contents.html' %}
+
+  {%- if content_block.guidance -%}
+    {% call onsPanel({
+      "id": "item-guidance-" ~ content_block.id,
+      "classes": "ons-u-mb-m"
+      }) %}
+      {% set contents = content_block.guidance.contents %}
+      {% include 'partials/contents.html' %}
+    {% endcall %}
+  {% endif %}
```

Comment:
The order should be okay. Author will have to make use of multiple elements if they want guidance in between.